File: Dev/Frontend/LambdaPP_WebApp_Alpha_3.0/WildTask/src/app/logic/models/Skill.ts
import Employee_Skill_Bridge from "./Employee_Skill_Bridge";

export default class Skill
{
    skill_ID: number | null;
    skill_Name: string;
    skill_Description: string;
    skill_Enabled: boolean;
    Employee_Skill_Bridges: Employee_Skill_Bridge[] | undefined;
    
    constructor();
    constructor(name?: string, description?: string)
    {
        this.skill_ID = null;
        if (name) this.skill_Name = name;
        else this.skill_Name = "";
        if (description) this.skill_Description = description;
        else this.skill_Description = "";
        this.skill_Enabled = true;
    }
}

File: Mobile/D2/LambdaPP_AndroidApp_D2/app/src/main/java/com/example/lambdapp_androidapp_d2/models/Skill.java
package com.example.lambdapp_androidapp_d2.models;

import java.util.List;

public class Skill {
    public int skill_ID;
    public String skill_Name;
    public String skill_Description;
    public boolean skill_Enabled;
    public List<Employee_Skill_Bridge> employee_Skill_Bridges;
    //public List<Employee> employees;
    public List<Task_Skill_Bridge> task_Skill_Bridges;
    //public List<Task>? tasks;
}
